repose-aggregator/components/filters/uri-normalization-filter/pom.xml

```diff
@@ -92,6 +92,7 @@
 
                     <args>
                         <arg>-Xfluent-api</arg>
+                        <arg>-XtoString</arg>
                     </args>
```

Comment:
I'm not sure why someone added the default-value arg to the gradle build.  I cleaned it up.